src/settings.ts

```diff
@@ -2,7 +2,7 @@ import * as vscode from 'vscode';
 import * as path from 'path';
 import { config } from './editor';
 
-const ReactVersion = require('child_process').execSync('npm list react --version --depth=0').toString().trim().split('.').map(Number);
+const ReactVersion = require('child_process').execSync('npm view react version').toString().trim().split('.').map(Number);
```

Comment:
Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/wix/vscode-glean/pullRequest?prid=5796206)

Reply:
Happy to change, but the original document uses single quotes. Not sure if this setting is intentional or a misconfig of Codacy.